cmd/git/git.go

```diff
@@ -1148,6 +1155,7 @@ func (j *DSGit) GetModelData(ctx *shared.Ctx, docs []interface{}) []git.CommitCr
 		commit.ShortHash, _ = doc["hash_short"].(string)
 		source, _ := doc["commit_repo_type"].(string)
 		commit.Message, _ = doc["message"].(string)
+		commit.Origin = git.CommitOrigin(source)
 		_, commit.Orphaned = j.OrphanedMap[commit.SHA]
```

Comment:
this is a new field to store commit source/origin - I don't want to update the base event `source` field because that one is for the data source name which is constant = "git" for this connector.

Reply:
Never mind, updated to @fayazg suggested approach.